JetSet/JetSet/WarthogDemoTodos.swift

```diff
@@ -101,15 +91,17 @@ struct WarthogDemoTodos: View {
                         todo.done.toggle()
                     }
                 }
+            } header: {
+                Text("Computer Systems")
             }
-            
-            
         }
-        
+        .navigationTitle("A-10 Procedures")
     }
 }
 
 #Preview {
-    WarthogDemoTodos()
+    NavigationStack {
+        WarthogDemoTodos()
```

Comment:
Using a `NavigationStack` in your preview lets you demonstrate navigation link behavior without needing to launch your app